prov/efa/test/efa_unit_test_ep.c

```diff
@@ -966,13 +967,22 @@ static void test_efa_rdm_ep_use_zcpy_rx_impl(struct efa_resource *resource,
 	assert_int_equal(fi_enable(resource->ep), 0);
 
 	assert_true(ep->use_zcpy_rx == expected_use_zcpy_rx);
+
+	assert_int_equal(fi_getopt(&resource->ep->fid, FI_OPT_ENDPOINT, FI_OPT_INJECT_RMA_SIZE,
+			&inject_msg_size, &(size_t){sizeof inject_msg_size}), 0);
```

Comment:
Intentional? (`FI_OPT_INJECT_RMA_SIZE` and `inject_msg_size`)

Reply:
Sorry, typo